lib/ansible/modules/network/ios/ios_lag_interfaces.py

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat Inc.
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+##############################################
+#                 WARNING
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#
+##############################################
+
+"""
+The module file for ios_l3_interfaces
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+
+DOCUMENTATION = """
+---
+module: ios_lag_interfaces
+version_added: 2.9
+short_description: Manage Link Aggregation on Cisco IOS devices.
+description: This module manages properties of Link Aggregation Group on Cisco IOS devices.
+author: Sumit Jaiswal (@justjais)
+notes:
+  - Tested against Cisco IOSv Version 15.2 on VIRL
+  - This module works with connection C(network_cli).
+    See L(IOS Platform Options,../network/user_guide/platform_ios.html).
+options:
+config:
+  description: A list of link aggregation group configurations.
+  type: list
+  elements: dict
+  suboptions:
+    name:
+      description:
+      - ID of Ethernet Channel of interfaces.
+      - Refer to vendor documentation for valid port values.
+      type: int
```

Comment:
are you sure that this is an int?, this is defined as str on your argspec `'name': {'required': True, 'type': 'str'},`